code/__HELPERS/game.dm

```diff
@@ -579,4 +579,4 @@ var/list/DummyCache = list()
 	//if(mixedcolor<0x00 || mixedcolor>0xFF)
 	//	return 0
 	// that's not the kind of operation we are running here, nerd
-	return clamp(round(mixedcolor), 0, 255)
+	return clamp(round(mixedcolor), 0, 255)
```

Comment:
Not only did you change nothing but whitespace in this file, but the whitespace change made is a bad one

Reply:
I swear VS Code did that, not me.

Reply:
I know but you still committed it
Also you set it to remove newlines at end of file, since there's no way it defaults to anything other than to *add* a newline at the end of a file